### PR TITLE
Add fingerprint plugin for GeoVision

### PR DESCRIPTION
**Pull request by ybadaoui-ostorlab**
Add fingerprint plugin for GeoVision

### CODE REVIEW

**Review by nmasdoufi-ol — APPROVED** [review]
LGTM!

**Review by benyissa — COMMENTED** [review]
LGTM

File: plugins/geovision.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    ]
+    version "0.1"
+    description "GeoVision specializes in advanced video surveillance solutions, offering state-of-the-art IP cameras, cloud-based surveillance platforms. etc..."
+    website "https://www.geovision.com.tw/"
+  
+    matches [
+      {:text => "<img src=\"img/WebLogin/logo.png\""},
+      {:regexp => /Geovision Inc./i},
+      {:text => "<title>Geovision"}
+    ]
```
Comment (mohsinenar) [inline plugins/geovision.rb:14]: There is nothing here aboutr the version!

**Review by mohsinenar — CHANGES_REQUESTED** [review]

File: plugins/geovision.rb (hunk 1 above)
Comment (ybadaoui-ostorlab) [reply plugins/geovision.rb:14]: This is plugin version, and this plugin is just for detecting the use of Geovision devices and not the version!

**Review by 3asm — APPROVED** [review]

**Pull request merged**